package.json

```diff
@@ -95,18 +93,16 @@
     "listr": "^0.13.0",
     "loader-utils": "^1.1.0",
     "lodash": "^4.17.5",
-    "log-symbols": "2.2.0",
+    "log-symbols": "^2.2.0",
     "mkdirp": "^0.5.1",
     "p-each-series": "^1.0.0",
     "p-lazy": "^1.0.0",
     "prettier": "^1.5.3",
-    "recast": "^0.14.4",
     "resolve-cwd": "^2.0.0",
     "supports-color": "^5.2.0",
-    "uglifyjs-webpack-plugin": "^1.2.2",
     "v8-compile-cache": "^1.1.2",
     "webpack-addons": "^1.1.5",
-    "yargs": "9.0.1",
+    "yargs": "^9.0.1",
```

Comment:
@shellscape which version was corrupted?

Reply:
I believe it was 8.x, but I can't recall for sure.